src/desktop/components/split_test/running_tests.coffee

```diff
@@ -45,4 +45,13 @@ module.exports = {
     weighting: 'equal'
     edge: 0
     control_group: 9
+
+  artist_insights:
+    key: 'artist_insights'
+    outcomes: [
+      'v1'
+      'v2'    ]
```

Comment:
weird spacing (since coffeescript is whitespace sensitive, might be better to err on caution and move the bracket